src/kernels/raw/launcher/kernelProcess.node.ts

```diff
@@ -367,7 +369,10 @@ export class KernelProcess implements IKernelProcess {
     }
     private async createConnectionFile() {
         const runtimeDir = await this.jupyterPaths.getRuntimeDir();
-        const tempFile = await this.fileSystem.createTemporaryLocalFile({ fileExtension: '.json', prefix: 'kernel-' });
+        const tempFile = await this.fileSystem.createTemporaryLocalFile({
+            fileExtension: '.json',
+            prefix: 'kernel-v2-'
```

Comment:
Added `v2` so if users report files aren't getting deleted we know whether its old or new stuff.
Also helps me figure out whether these are before or after this change